### PR TITLE
BETA: Register mariano.avila.is-a.dev

### PR DESCRIPTION
**Pull request by DxRavage**
Added `mariano.avila.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/m.avila.json
Comment (Stef-00012) [inline domains/m.avila.json:1]: Invalid file name (each subdomain must be 2 characters long)

File: domains/m.avila.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "DxRavage",
+        "email": "marianoavilaj4@gmail.com"
+    },
+    "record": {
+        "CNAME": "www.m.avila.is-a.dev"
```
Comment (Stef-00012) [inline domains/m.avila.json:7]: You can't point a CNAME to the domain itself

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/m.avila.json
Comment (DxRavage) [reply domains/m.avila.json:1]: {
    "description": "personal portfolio",
    "repo": "https://github.com/dxravage/dxravage.github.io",
    "owner": {
        "username": "dxravage",
        "email": "marianoavilaj4@gmail.com",
        "twitter": "MarianoAvilaC"
    },
    "record": {
        "CNAME": "dxravage.github.io"
    }

**Review by DxRavage — COMMENTED** [review]
my bad updated following guideliness in documentation

**Comment by Stef-00012** [comment]
Now your pr is empty (0 files changed) so i'll close it

**Pull request closed without merging**